### PR TITLE
Octo logging

### PR DESCRIPTION
**Pull request by bbaldino**
when i was trying to debug some octo issues i found myself wishing we had some extra logs to make things clear.  i suspect we may be removing a relay handler prematurely (something that could be coming from the shims which have changes in 2.0, so would explain why we're not seeing it in old bridge)

### CODE REVIEW

File: src/main/java/org/jitsi/videobridge/octo/OctoRelay.java [hunk 1]
```diff
@@ -19,7 +19,8 @@
 import org.jitsi.rtp.*;
 import org.jitsi.rtp.rtp.*;
 import org.jitsi.utils.*;
-import org.jitsi.utils.logging.*;
+import org.jitsi.utils.collections.*;
+import org.jitsi.utils.logging2.*;
```
Comment (JonathanLennox) [inline src/main/java/org/jitsi/videobridge/octo/OctoRelay.java]: Huh, we still had old logger in jvb in some places? I thought you'd switched everything over.  Are there any other places like this?

File: src/main/java/org/jitsi/videobridge/octo/OctoTentacle.java [hunk 2]
```diff
@@ -312,8 +312,9 @@ public void addRtpExtension(RtpExtension rtpExtension)
      */
     public void expire()
     {
+        logger.info("Expiring");
```
Comment (JonathanLennox) [inline src/main/java/org/jitsi/videobridge/octo/OctoTentacle.java]: This message is a bit terse.  Add what you're expiring?

File: src/main/java/org/jitsi/videobridge/octo/OctoRelay.java (hunk 1 above)
Comment (bbaldino) [reply src/main/java/org/jitsi/videobridge/octo/OctoRelay.java]: yes, looks like there are.  i'll do a final sweep of this soon.

File: src/main/java/org/jitsi/videobridge/octo/OctoTentacle.java (hunk 2 above)
Comment (bbaldino) [reply src/main/java/org/jitsi/videobridge/octo/OctoTentacle.java]: it's terse just because the log context will already give the other info

Comment (JonathanLennox) [reply src/main/java/org/jitsi/videobridge/octo/OctoTentacle.java]: I forget, does log context show up even if someone has the "wrong" log formatter configured?  I forget the order of operations.

Comment (bbaldino) [reply src/main/java/org/jitsi/videobridge/octo/OctoTentacle.java]: no, that is done in the formatter, but we've already done this in some other places and, although i want to make sure things don't 'break' with old formatter i'm not really feeling like fully-accommodating it either, as we should just be migrating.  more motivation :)

Comment (JonathanLennox) [reply src/main/java/org/jitsi/videobridge/octo/OctoTentacle.java]: Fair enough.

**Pull request merged**